### PR TITLE
pass correct arguments to logging.info

### PR DESCRIPTION
**Pull request by murtraja**

### CODE REVIEW

**Comment by murtraja** [comment]
Fix for #49

File: download.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def get_correctly_labeled_entries(all_entries):
 
 
 def download_csv_dataset(csv_url, dataset_csv_path):
-    logging.info("Downloading CSV from ", csv_url, " to ", dataset_csv_path, ".")
+    logging.info("Downloading CSV from %s to %s.", csv_url, dataset_csv_path)
```
Comment (piraka9011) [inline download.py:50]:
```suggestion
    logging.info(f"Downloading CSV from {csv_url} to {dataset_csv_path}.")
```
We're moving to using f-strings in all the codebase.

Comment (piraka9011) [reply download.py]: Or use the `.format()` method.

**Review by piraka9011 — APPROVED** [review]
Awesome JAK for the fix!
If you can change the string formatting to use the `.format()` method or f-strings that'd be great :)

**Pull request merged**